apollo-backend/src/main/java/io/logz/apollo/controllers/StatusController.java

```diff
@@ -249,13 +249,14 @@ public String getLatestPodName(int environmentId, int serviceId, String groupNam
         return serviceStatusHandler.getUndeployedServicesByAvailability(availability, timeUnitEnum, duration);
     }
 
-    private KubernetesDeploymentStatus addCommitUrlToStatus(KubernetesDeploymentStatus kubernetesDeploymentStatus, int serviceId) {
+    private String getCommitUrl(KubernetesDeploymentStatus kubernetesDeploymentStatus, int serviceId) {
+        String gitCommitUrl = null;
         if (kubernetesDeploymentStatus != null && kubernetesDeploymentStatus.getGitCommitSha() != null) {
             DeployableVersion deployableVersion = deployableVersionController.getDeployableVersionFromSha(kubernetesDeploymentStatus.getGitCommitSha(), serviceId);
             if (deployableVersion != null) {
-                kubernetesDeploymentStatus.setGitCommitUrl(deployableVersion.getCommitUrl());
+                gitCommitUrl = deployableVersion.getCommitUrl();
             }
         }
-        return kubernetesDeploymentStatus;
+        return gitCommitUrl;
```

Comment:
Ok to return null?

Reply:
Yes, it just means that the link wouldn't work.